src/lib/gist-handler.js

```diff
@@ -12,11 +12,13 @@ function GistHandler (_window) {
     var gistId
     if (params['gist'] === '') {
       loadingFromGist = true
-      modalDialogCustom.prompt(null, 'Enter the ID of the Gist you would like to load.', null, (target) => {
+      modalDialogCustom.prompt(null, 'Enter the ID of the Gist or URL you would like to load.', null, (target) => {
```

Comment:
not sure `URL` works in that case, are you sure of this change?

Reply:
works for me